analyses/bangladesh/trigger_development/read_in_data.py

```diff
@@ -0,0 +1,173 @@
+import sys
+import os
+from pathlib import Path
+
+import matplotlib as mpl
+from scipy.stats import norm, pearsonr
+import numpy as np
+import pandas as pd
+import xarray as xr
+
+path_mod = f"{Path(os.path.dirname(os.path.realpath(''))).parents[1]}/"
+sys.path.append(path_mod)
+
+from src.indicators.flooding import glofas
+from src.bangladesh import get_glofas_data as ggd
```

Comment:
Out of curiosity, what do you think the value is in storing the global variables for BGD in ```get_glofas_data``` vs this script itself?

Reply:
Good question, so at the moment `get_glofas_data` is kind of acting as the main entry script, and thus seems like the most natural place to store this information, which probably shouldn't be unnecessarily duplicated. But anyway with the Nepal flooding PR I refactored the GloFAS class so I don't think these variables will be required anymore.